app/src/sharedTest/java/org/oppia/app/profile/AdminAuthActivityTest.kt

```diff
@@ -132,6 +135,87 @@ class AdminAuthActivityTest {
     }
   }
 
+  @Test
+  fun testAdminAuthActivity_buttonState_ConfigurationChanged_ButtonStateIsPreserved() {
```

Comment:
The name should be in this way:

`testAdminAuthActivity_buttonState_configurationChanged_buttonStateIsPreserved`

the first character after `_` should be small

Reply:
Done